sycl/include/CL/sycl/ONEAPI/intel_matrix/matrix.hpp

```diff
@@ -0,0 +1,16 @@
+//==------------------ matrix.hpp - SYCL matrix ----------------*- C++ -*---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// ===--------------------------------------------------------------------=== //
+/// -DAMX will enable joint_matrix feature for AMX
```

Comment:
We don't need -DAMX anymore. Please update.
Also, what should be the compilation command line the user has to use to trigger this extension:
clang++ -fsycl -march=sapphirerapids 
Is it this one?